ackhandler/sent_packet_handler.go

```diff
@@ -333,7 +333,11 @@ func (h *sentPacketHandler) SendingAllowed() bool {
 			h.bytesInFlight,
 			h.congestion.GetCongestionWindow())
 	}
-	return !(congestionLimited || maxTrackedLimited)
+	// Workaround for #555:
+	// Always allow sending of retransmissions. This should probably be limited
+	// to RTOs, but we currently don't have a nice way of distinguishing them.
```

Comment:
Why is this only a workaround? I thought this is (part of) the solution, to always send out RTOs, no matter what the congestion controller says.